src/reference/interactive/index.pug

```diff
@@ -177,72 +184,81 @@ block reference
             td #[a(href='https://watchbeam.github.io/beam-interactive-node2' target='_blank') Reference Docs]
-            td Great for Interactive tools and utilities and beginners.
+            td Great for creating Interactive tools and utilities. Suitable for beginners.
         tr
             td C++
             td #[a(href='https://github.com/WatchBeam/interactive-sdk-cpp' target='_blank') GitHub]
             td #[a(href='/reference/interactive/cplusplus/index.html' target='_blank') Reference Docs]
-            td Integrate directly into a C++ Game.
+            td Add Beam Interactive features directly into a C++ Game.
         tr
             td Unity
             td #[a(href='https://github.com/WatchBeam/interactive-unity-plugin/releases/tag/v0.1.0-alpha') GitHub]
             td
                 span #[a(href='/reference/interactive/csharp/index.html' target='_blank') Reference Docs]
                 br
                 span #[a(href='https://github.com/WatchBeam/interactive-unity-plugin/wiki/Getting-started' target='_blank') Getting Started]
-            td Prototype Quickly or integrate into a Unity based Game.
+            td Add Beam Interactive features into a game created using Unity. Good for quick prototyping.
 
-    p Once you've chosen an SDK from the above list, download it from its repository and get it setup in your favorite IDE and ready to go.
+    p Download the appropriate SDK that your project needs from the repository and create a new project in your favorite IDE.
 
-    h3#creating-an-interactive-project Creating an Interactive Project
-    p Interactive projects are created and configured in the #[a(href='https://beam.pro/i/studio' target='_blank') Interactive Studio]. To create a new project visit the #[a(href='https://beam.pro/i/studio' target='_blank') studio] and click on the plus button:
+    h3#creating-an-interactive-project Create a new Interactive Project
+    
+    p Interactive projects are created and configured in the #[a(href='https://beam.pro/i/studio' target='_blank') Beam Interactive Studio]. 
+    
+    
+    p Go to #[a(href='https://beam.pro/i/studio' target='_blank') Studio] and click on the #[strong plus] button to create a new project.
 
     .figure
         img(src='/img/reference/interactive/studio/createNewProject.png' alt='Screenshot showing the location of the create new project button')
-    p.caption Screenshot showing the location of the create new project button.
+    p.caption Screenshot shows the location of the Create new project button.
 
-    p You'll then be prompted to enter a name for your project. Once a name is entered you'll be taken to the Project Editor.
+    p You'll then be prompted to enter a name for your project. Enter a suitable name for your project.
+    
+    p After creating the project, you'll be taken to the project editor. You can still modify your project using the editor.
```

Comment:
"You can still modify your project using the editor." doesnt make sense, we havent left the editor why is the extra sentence needed